migrations/sql/V2022.06.23.09.30__create_notice_of_departure_contacts_table.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE IF NOT EXISTS notice_of_departure_contact
+(
+    notice_of_departure_contact_guid uuid DEFAULT gen_random_uuid() PRIMARY KEY,
+    notice_of_departure_guid         uuid NOT NULL,
```

Comment:
```suggestion
    nod_guid         uuid NOT NULL,
```
for the sake of consistency